mdk/build.gradle

```diff
@@ -81,19 +80,10 @@ minecraft {
         gameTestServer {
             workingDirectory project.file('run')
 
-            // Recommended logging data for a userdev environment
-            // The markers can be added/remove as needed separated by commas.
-            // "SCAN": For mods scan.
-            // "REGISTRIES": For firing of registry events.
-            // "REGISTRYDUMP": For getting the contents of all registries.
             property 'forge.logging.markers', 'REGISTRIES'
 
-            // Recommended logging level for the console
-            // You can set various levels here.
-            // Please read: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
             property 'forge.logging.console.level', 'debug'
 
-            // Comma-separated list of namespaces to load gametests from. Empty = all namespaces.
```

Comment:
Why were all the comments removed?
Yes they are verbose, but they are there to direct people where to look.

Reply:
Because they're duplicates of the same comments found in the client run config just above

Reply:
ah, works for me, once the pack format is resolved this looks pullable.

Reply:
The pack.mcmeta is already resolved. <https://github.com/MinecraftForge/MinecraftForge/blob/ade2b8cc80254f050b38ff3d32655f08b259a21a/mdk/src/main/resources/pack.mcmeta>